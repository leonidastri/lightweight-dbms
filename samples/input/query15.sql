select DISTINCT * from Sailors, Boats, Reserves WHERE Boats.D < Reserves.H;