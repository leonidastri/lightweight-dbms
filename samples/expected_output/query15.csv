1,200,50,101,2,3,1,102
2,200,200,101,2,3,1,102
1,200,50,101,2,3,1,103
1,200,50,102,3,4,1,103
2,200,200,101,2,3,1,103
2,200,200,102,3,4,1,103
1,200,50,101,2,3,3,102
2,200,200,101,2,3,3,102
1,200,50,101,2,3,4,104
1,200,50,102,3,4,4,104
1,200,50,103,1,1,4,104
2,200,200,101,2,3,4,104
2,200,200,102,3,4,4,104
2,200,200,103,1,1,4,104
3,100,105,101,2,3,1,102
4,100,50,101,2,3,1,102
3,100,105,101,2,3,1,103
3,100,105,102,3,4,1,103
4,100,50,101,2,3,1,103
4,100,50,102,3,4,1,103
3,100,105,101,2,3,3,102
4,100,50,101,2,3,3,102
3,100,105,101,2,3,4,104
3,100,105,102,3,4,4,104
3,100,105,103,1,1,4,104
4,100,50,101,2,3,4,104
4,100,50,102,3,4,4,104
5,100,500,101,2,3,1,102
6,300,400,101,2,3,1,102
5,100,500,101,2,3,1,103
5,100,500,102,3,4,1,103
6,300,400,101,2,3,1,103
6,300,400,102,3,4,1,103
5,100,500,101,2,3,3,102
6,300,400,101,2,3,3,102
4,100,50,103,1,1,4,104
5,100,500,101,2,3,4,104
5,100,500,102,3,4,4,104
5,100,500,103,1,1,4,104
6,300,400,101,2,3,4,104
6,300,400,102,3,4,4,104
6,300,400,103,1,1,4,104